File: 프로그래머스/unrated/181837. 커피 심부름/커피 심부름.java
//1. 기본 풀이.

// class Solution {
//     public int solution(String[] order) {
//         int answer = 0;
//         for(String offer : order)
//             answer += offer.contains("cafelatte") ? 5000 : 4500;    
//         return answer;
//     }
// }

//2. 스트림 풀이.
import java.util.Arrays;
class Solution {
    public int solution(String[] order) {    
    return Arrays.stream(order).mapToInt(offer->offer.contains("cafelatte") ? 5000 : 4500).sum();
    }
}

File: 프로그래머스/unrated/181837. 커피 심부름/커피 심부름.cs
using System;

public class Solution {
    public int solution(string[] order) {
        
        int cnt = 0;
        int size = order.Length;
        for(int i = 0; i < size; i++){
            if(order[i].IndexOf("cafelatte") >= 0) cnt ++;
        }
        return cnt * 5000 + (size - cnt) * 4500;
    }
}